eng/packages/http-client-csharp/generator/Azure.Generator/src/Providers/Abstraction/HttpPipelineProvider.cs

```diff
@@ -39,7 +39,7 @@ public override ClientPipelineApi FromExpression(ValueExpression expression)
             => new HttpPipelineProvider(expression);
 
         public override ValueExpression PerRetryPolicy(params ValueExpression[] arguments)
```

Comment:
The args that are passed in from MGC.CM are specific to authorization. I think it would be better to name this as the AuthorizationPolicy. We may have a need to allow more generic PerRetry/PerCall policies in the future, but this is a very specific policy.

Reply:
Right. The method name is not generic enough to handle all policies.
How about change the signature in MGC to
```csharp
public override ValueExpression ApplyAuthorizationPolicies(params ValueExpression[] authorizationPolicies)
```

Reply:
Well this isn't applying the policies, it is just asking for the authorization policy and then MGC will generate the code to add it to the pipeline.

Reply:
Renamed in https://github.com/microsoft/typespec/pull/5105
Will apply the latest version of MGC once it's merged

Reply:
applied